test/benchmark/test-benchmark-http.js

```diff
@@ -29,7 +29,8 @@ runBenchmark('http',
                'n=1',
                'res=normal',
                'type=asc',
-               'value=X-Powered-By'
+               'value=X-Powered-By',
+               'headerDuplicates=1',
```

Comment:
Can you relocate it higher in the list to keep the list in alphabetical order?

Reply:
> Can you relocate it higher in the list to keep the list in alphabetical order?

Argh, never mind, it's already not alphabetical, just mostly alphabetical. Anyway, LGTM, but nit: Would prefer to alphabetize the list.

(Also, agree that `n` is probably better. The headers aren't duplicates. They're similarly-named.)